SQL Scripts/CreateDB.sql

```diff
@@ -31,7 +31,7 @@ CREATE TABLE Messages (
 );
 
 CREATE TABLE Photo (
-    image BYTEA NOT NULL,
+    image varchar(100) NOT NULL,
```

Comment:
[Best database field type for a url](https://stackoverflow.com/questions/219569/best-database-field-type-for-a-url)

Recommended to set VARCHAR to 2083